Dockerfile.in

```diff
@@ -0,0 +1,48 @@
+FROM php:PHPVER-apache
+MAINTAINER Martin Helmich <typo3@martin-helmich.de>
+
+# Install TYPO3
+RUN apt-get update &&\
+    apt-get install -y --no-install-recommends \
+        wget \
+# Configure PHP
+        libxml2-dev libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libmcrypt-dev \
+        libpng12-dev \
+        zlib1g-dev \
+# Install required 3rd party tools
+        graphicsmagick && \
+    docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && \
+    docker-php-ext-install -j$(nproc) mysqli soap gd zip opcache && \
+    echo 'always_populate_raw_post_data = -1\nmax_execution_time = 240\nmax_input_vars = 1500\nupload_max_filesize = 32M\npost_max_size = 32M' > /usr/local/etc/php/conf.d/typo3.ini && \
+# Configure Apache as needed
+    a2enmod rewrite && \
+    echo
+RUN    apt-get clean && \
```

Comment:
Is there any particular reason why there's a second `RUN` statement at this point? Clearing the `/var/lib/apt` directory might be much more efficient when done in the same build step as _all_ of the `apt-get install` commands.